Mathlib/Order/Filter/AtTopBot/Group.lean

```diff
@@ -13,115 +13,135 @@ import Mathlib.Order.Filter.AtTopBot.Monoid
+    Tendsto (fun x => f x * g x) l atBot :=
+  @tendsto_atTop_mul_right_of_le' _ Gᵒᵈ _ _ _ _ C hf hg
+
+@[to_additive]
+theorem tendsto_atTop_mul_right_of_le (C : G) (hf : Tendsto f l atTop) (hg : ∀ x, C ≤ g x) :
+    Tendsto (fun x => f x * g x) l atTop :=
+  tendsto_atTop_mul_right_of_le' l C hf (univ_mem' hg)
+
+@[to_additive]
+theorem tendsto_atBot_mul_right_of_ge (C : G) (hf : Tendsto f l atBot) (hg : ∀ x, g x ≤ C) :
+    Tendsto (fun x => f x * g x) l atBot :=
+  @tendsto_atTop_mul_right_of_le _ Gᵒᵈ _ _ _ _ C hf hg
+
+@[to_additive]
+theorem tendsto_atTop_mul_const_left (C : G) (hf : Tendsto f l atTop) :
+    Tendsto (fun x => C * f x) l atTop :=
+  tendsto_atTop_mul_left_of_le' l C (univ_mem' fun _ => le_refl C) hf
+
+@[to_additive]
+theorem tendsto_atBot_mul_const_left (C : G) (hf : Tendsto f l atBot) :
+    Tendsto (fun x => C * f x) l atBot :=
+  @tendsto_atTop_mul_const_left _ Gᵒᵈ _ _ _ C hf
+
+@[to_additive]
+theorem tendsto_atTop_mul_const_right (C : G) (hf : Tendsto f l atTop) :
+    Tendsto (fun x => f x * C) l atTop :=
+  tendsto_atTop_mul_right_of_le' l C hf (univ_mem' fun _ => le_refl C)
+
+@[to_additive]
+theorem tendsto_atBot_mul_const_right (C : G) (hf : Tendsto f l atBot) :
+    Tendsto (fun x => f x * C) l atBot :=
+  @tendsto_atTop_mul_const_right _ Gᵒᵈ _ _ _ C hf
+
+@[to_additive]
+theorem map_inv_atBot : map (Inv.inv : G → G) atBot = atTop :=
+  (OrderIso.inv G).map_atBot
+
+@[to_additive]
+theorem map_inv_atTop : map (Inv.inv : G → G) atTop = atBot :=
+  (OrderIso.inv G).map_atTop
+
+@[to_additive]
+theorem comap_inv_atBot : comap (Inv.inv : G → G) atBot = atTop :=
+  (OrderIso.inv G).comap_atTop
+
+@[to_additive]
+theorem comap_inv_atTop : comap (Inv.inv : G → G) atTop = atBot :=
+  (OrderIso.inv G).comap_atBot
+
+@[to_additive]
+theorem tendsto_inv_atTop_atBot : Tendsto (Inv.inv : G → G) atTop atBot :=
+  (OrderIso.inv G).tendsto_atTop
+
+@[to_additive]
+theorem tendsto_inv_atBot_atTop : Tendsto (Inv.inv : G → G) atBot atTop :=
+  @tendsto_inv_atTop_atBot Gᵒᵈ _
 
 variable {l}
 
-@[simp]
-theorem tendsto_neg_atTop_iff : Tendsto (fun x => -f x) l atTop ↔ Tendsto f l atBot :=
-  (OrderIso.neg β).tendsto_atBot_iff
+@[to_additive (attr := simp)]
+theorem tendsto_inv_atTop_iff : Tendsto (fun x => (f x)⁻¹) l atTop ↔ Tendsto f l atBot :=
+  (OrderIso.inv G).tendsto_atBot_iff
 
-@[simp]
-theorem tendsto_neg_atBot_iff : Tendsto (fun x => -f x) l atBot ↔ Tendsto f l atTop :=
-  (OrderIso.neg β).tendsto_atTop_iff
+@[to_additive (attr := simp)]
+theorem tendsto_inv_atBot_iff : Tendsto (fun x => (f x)⁻¹) l atBot ↔ Tendsto f l atTop :=
+  (OrderIso.inv G).tendsto_atTop_iff
 
-end OrderedGroup
+end OrderedCommGroup
 
-section LinearOrderedAddCommGroup
+section LinearOrderedCommGroup
 
-variable [LinearOrderedAddCommGroup α]
+variable [LinearOrderedCommGroup G]
 
 /-- $\lim_{x\to+\infty}|x|=+\infty$ -/
-theorem tendsto_abs_atTop_atTop : Tendsto (abs : α → α) atTop atTop :=
-  tendsto_atTop_mono le_abs_self tendsto_id
+@[to_additive]
```

Comment:
Here and below, should we copy the docstrings to the additive declaration as well?

Otherwise LGTM. Feel free to merge.
Thanks!
bors d+